packages/matchbox/src/components/QueryCard/Group.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledWrapper = styled.div`
+  & > div:not(:first-child) {
+    margin-top: -1px;
+    border-top-right-radius: 0;
+    border-top-left-radius: 0;
+  }
+
+  & > div:first-child {
```

Comment:
Could add `:not(:only-child)` here just so that if a group is created programmatically and ends up with only one card the styles aren't wrong on the single `QueryCard`